lib/src/cards.dart

```diff
@@ -402,8 +413,10 @@ class TCardState extends State<TCard> with TickerProviderStateMixin {
       _runChangeOrderAnimation();
       if (isSwipeLeft) {
         _swipeInfoList.add(SwipeInfo(_frontCardIndex, SwipeDirection.Left));
+        _backCallback();
```

Comment:
I think this is breaking compared to the original package, as _backCallback() was used for reversing the stack IIRC

Reply:
Mmh, maybe 

But it's strange, they use the backward to signal a swipe left, but also reverse the stack. Which are two different thing imho